utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -525,6 +544,16 @@ void PessimisticTransactionDB::UnregisterTransaction(Transaction* txn) {
   transactions_.erase(it);
 }
 
+Status WritePreparedTxnDB::Get(const ReadOptions& options,
+                               ColumnFamilyHandle* column_family,
+                               const Slice& key, PinnableSlice* value) {
+  SequenceNumber last_seq = db_impl_->GetLatestSequenceNumber();
```

Comment:
That is a good point. We do not need to do that since ReadCallback already take care of it. But we should clarify that so these two features do not get mixed up.

Reply:
Can you at leave a comment here to clarify the correctness? Since GetImpl also take a snapshot which can be >= `last_seq` here, the logic seems confusing but also seems no correctness issue.

Reply:
Sure. will do.

Reply:
Not sure if it is a real issue and if so can be address later. Since we are not taking explicit snapshot (i.e. db->GetSnapshot()), will there be chance if a key with prep_seq < last_seq < commit_seq get removed from commit map before IsInSnapshot is called?

Reply:
That is a great point. In theory it could actually cause returning NotFound. I change it to use kMaxSeq instead of LastSeq.